ecommerce/colaraz_features/api/helpers.py

```diff
@@ -0,0 +1,61 @@
+"""
+Helpers for colaraz API.
+"""
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+from oscar.core.loading import get_model
+
+from ecommerce.core.models import SiteConfiguration
+from ecommerce.theming.models import SiteTheme
+
+Partner = get_model('partner', 'Partner')
+
+def get_or_create_site(site_url):
+    site, _ = Site.objects.get_or_create(
+        domain=site_url,
+        name=site_url,
+    )
```

Comment:
I think the correct syntax is

```
        site, _ = Site.objects.get_or_create(
            domain=site_domain,
            defaults={'name': site_url}
        )
```
https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/theming/management/commands/create_sites_and_configurations.py#L107

Reply:
It doesn't matter in this case, however updating that.